src/spec.c

```diff
@@ -1017,6 +1215,21 @@ static int parseVectorField(IndexSpec *sp, StrongRef sp_ref, FieldSpec *fs, Args
     // Point to the same logCtx as the external wrapping VecSimParams object, which is the owner.
     params->logCtx = logCtx;
     result = parseVectorField_hnsw(fs, params, ac, status);
+  } else if (STR_EQCASE(algStr, len, VECSIM_ALGORITHM_SVS)) {
+    fs->vectorOpts.vecSimParams.algo = VecSimAlgo_TIERED;
+    VecSim_TieredParams_Init(&fs->vectorOpts.vecSimParams.algoParams.tieredParams, sp_ref);
+    fs->vectorOpts.vecSimParams.algoParams.tieredParams.specificParams.tieredSVSParams.trainingTriggerThreshold = 0; // Will be set to default value.
+
+    // primary index params allocated in VecSim_TieredParams_Init()
+    TieredIndexParams *params = &fs->vectorOpts.vecSimParams.algoParams.tieredParams;
+
+
+    // Forcibly set all SVS params to zero
+    memset(params->primaryIndexParams, 0, sizeof(VecSimParams));
+
```

Comment:
Let's set the default svs-vamana parameters here as we do for HNSW, so we can have those values set to return them in info. This will require defining them in vecsim